src/zodbbrowser/check.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import logging
+import optparse
+import os
+import sys
+import tempfile
+
+from zodbbrowser.standalone import open_database
+from zodbbrowser.references import ReferencesDatabase
+
+
+def iter_database(db):
+    """Iter over records located inside the database.
+    """
+    for transaction in db.storage.iterator():
+        for record in transaction:
+            yield record
+
+
+def main(args=None):
+    logging.basicConfig(format="%(message)s")
+
+    if args is None:
+        args = sys.argv[1:]
+    else:
+        args = args + sys.argv[1:]
```

Comment:
An interesting choice.  I'm used to `main(args=...)` *replacing* sys.argv, for test purposes.